gensim/models/word2vec.py

```diff
@@ -746,6 +753,9 @@ def job_producer():
             job_batch, batch_size = [], 0
             pushed_words, pushed_examples = 0, 0
             next_alpha = self.alpha
+            if next_alpha > self.min_alpha_yet_reached:
```

Comment:
This looks like a good minimal catch of the problem situation! Are you able to trigger this, for example by calling `train()` a second time?

The point of this error message is just to make the user think about whether their re-training (without the usual descending-alpha of typical SGD) makes sense. So let's word it: "Effective 'alpha' higher than previous training cycles. Is this your intent?"